### DISCUSSION TITLE
Condition Triggers

### DISCUSSION THREAD

**Discussion by FuriousMeatballs** [DFreds/dfreds-convenient-effects#300] [Ideas]
I want to make the full jump to convenient effects but one of the things I really like about Condition Lab and Triggler is the ability to set triggers to apply the conditions. Especially for conditions like unconscious when a player's hp falls to 0. Any plan for something like that?

**Comment by thatlonelybugbear** [comment]
Are you maybe using MidiQOL module? If yes, the interaction you describe is already automated. Just my quick 2 cents :)

**Reply by FuriousMeatballs** [reply]
Ah I see the option now lol I seemed to have missed it. Thank you @thatlonelybugbear for pointing that out. Still would be handy to set our own triggers for status though!

**Reply by DFreds** [reply]
Personally, I think that falls outside the scope of CE itself. I try to avoid too much bloat in my modules in order to keep them tightly focused on the thing they are doing. Creating a separate module that leverages CE's comprehensive API is certainly a possibility, and one I could see taking on in the future. But it won't be added to CE itself
